python/cudf/cudf/io/parquet.py

```diff
@@ -299,103 +296,6 @@ def _process_dataset(
     )
 
 
-def _get_byte_ranges(file_list, row_groups, columns, fs, **kwargs):
```

Comment:
This logic is now handled in `fsspec.parquet`. I'd rather not maintain a redundant copy here :)